src/components/snackbar/examples/snackbar-positioning.scss

```diff
@@ -0,0 +1,5 @@
+limel-snackbar {
```

Comment:
Should we add the default `0` for `--snackbar-right` here, to be more clear?

Reply:
it's already inherited from the main component. My expectation is that the consumer knows CSS and is aware of what they are doing. So they only override what they have to override.